### PR TITLE
Deprecated and Refactored stop()

### PR DESCRIPTION
**Pull request by alexbowers**
- Deprecated from:
   - Object
   - StaticObject

- Refactored into:
   - Command
   - ErrorHandler

### CODE REVIEW

File: core/Object.php [hunk 1]
```diff
@@ -285,10 +285,16 @@ protected static function _parents() {
 	/**
 	 * Exit immediately. Primarily used for overrides during testing.
 	 *
+	 * @deprecated This method will be removed in a future version. Please use
```
Comment (davidpersson) [inline core/Object.php]: There's one blank line too much. Also I think you can remove all the double \\ for a single \

File: core/StaticObject.php [hunk 2]
```diff
@@ -164,13 +164,18 @@ protected static function _parents() {
 	/**
 	 * Exit immediately. Primarily used for overrides during testing.
 	 *
+	 * @deprecated This method will be removed in a future version. Please use
```
Comment (davidpersson) [inline core/StaticObject.php]: Same here.

**Comment by davidpersson** [comment]
Looks good to me. Fails only on PHP 7.0.0, succeeds on nightly.

**Comment by davidpersson** [comment]
Although we're hardly scraping past the limits of the [Rule of Three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming) I think this is pragrammatic enough.

**Comment by shopblocks** [comment]
Any update on getting these merged @davidpersson

**Comment by alexbowers** [comment]
@davidpersson I'm guessing this is dead in the water?

**Comment by davidpersson** [comment]
@alexbowers No it's not. I have a local branch containing these changes and more to Object. The refactor will be merged into 1.2. 1.1 has enough changes already plus the Object-refactor is not required for PHP 7 compatibility.

**Comment by davidpersson** [comment]
Give me some time to re-organize that branch. I'll ping you on this PR, if we can continue working on it.

**Comment by davidpersson** [comment]
Turns I said the same in https://github.com/UnionOfRAD/lithium/issues/1251 already. I might simply not understand what you're asking.

**Comment by davidpersson** [comment]
This is now part of the https://github.com/UnionOfRAD/lithium/tree/struct/deprecate-object branch and will be merged once it is complete.

**Pull request closed without merging**